core/src/docs/rfcs/6209_glob_support.md

```diff
@@ -69,36 +69,29 @@ The implementation would involve:
 
 1. Implementing a pattern matching logic for glob expressions. This can be a simplified version focusing on common use cases like `*`, `?`, and `**`.
 
-2. Modifying the `Lister` and `BlockingLister` (or their underlying streams/iterators) to accept a glob pattern and filter entries accordingly.
+2. Modifying the `List` and `Lister` to accept a glob pattern and filter entries accordingly.
```

Comment:
We don't have a struct named `List`. Additionally, `Lister` shouldn't receive a glob pattern directly; the pattern should be processed before initiating the actual list operation. Specifically, within the `list_with()` API.

Reply:
> so this glob api is on `FunctionList` ?

Actually, it's in `opendal::raw::OpList`; all other places simply expose it.